_posts/en/newsletters/2022-02-23-newsletter.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'Bitcoin Optech Newsletter #188'
+permalink: /en/newsletters/2022/02/23/
+name: 2022-02-23-newsletter
+slug: 2022-02-23-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Fee bumping and transaction fee sponsorship:** separate from the
+  replace-by-fee discussion started a couple weeks ago (see [Newsletter
+  #186][news186 rbf]), this week James O'Beirne [started][obeirne bump]
+  a discussion about fee bumping.  In particular, O'Beirne is concerned
+  that some of the transaction relay policy changes being proposed will
+  complicate the use of fee bumping for users and wallet developers.  As
+  an alternative, he seeks renewed consideration of [transaction fee
+  sponsorship][topic fee sponsorship] (previously described in
+  [Newsletter #116][news116 sponsorship]).
+
+    The idea received significant discussion on the mailing list.
```

Comment:
I appreciate the effort to be objective, I probably would have said "significant pushback"